### ISSUE TITLE
address potential abuse of Particle subclasses 

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/gas-properties#143]
In #111 (code review), @jbphet added the `REVIEW` comment shown below to `HeavyParticle`. A similar comment was added to `LightParticle`.  These particles types have fixed options that should not be overridden.

This also applies to `DiffusionParticle1` and `DiffusionParticle2`, although they have mutable mass and radius, so those options are fair game to override.

I'll investigate a better way to instantiate particles.

```js
class HeavyParticle extends Particle {

    // REVIEW: This seems like it has potential to be abused, such as passing in options that would make this "Heavy"
    // particle lighter than LightParticle.  As far as I could tell, the code doesn't vary these options, at least not
    // all of them.  Perhaps a factory function on the Particle class would be better.
    /**
     * @param {Object} [options] see Particle
     */
    constructor( options ) {
      super( _.extend( {

        // superclass options
        mass: 28, // equivalent to N2 (nitrogen), in AMU, rounded to the closest integer
        radius: GasPropertiesConstants.HEAVY_PARTICLES_RADIUS, // pm
        colorProperty: GasPropertiesColorProfile.heavyParticleColorProperty,
        highlightColorProperty: GasPropertiesColorProfile.heavyParticleHighlightColorProperty
      }, options ) );
    }
  }
```

**Comment by pixelzoom** [comment]
I did not use a factory method approach to solve this problem.  That felt like more of a change than I wanted to make at this late in development, and I'm partial to the class hierarchy approach in this case.   So here's what I did in the above commits...

For `HeavyParticle` and `LightParticle`, it was unnecessary to provide options to the constructor, so I simply removed the `options` parameter.

For `DiffusionParticle1` and `DiffusionParticle2`, I added assertions for the superclass options that should not be changed (colors). Other options are fair game, since mass and radius of these particle species is mutable.  The assertions use the same pattern employed in `BooleanProperty`.

Back to @jbphet for review.
